pkg/controllers/testutil/zcmd/zpool/fake.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package zpool
+
+import (
+	internalapi "github.com/openebs/api/pkg/internalapis/apis/cstor"
+)
+
+// MockPoolInfo contains the pool information which
+// will helpful to return error
+type MockPoolInfo struct {
```

Comment:
So the story is like this : 
Think of this struct as an object that helps you in mocking zpool command ? 
This object can be configured to replicate different zpool commands and outputs.
Who is this object? On of the answer could be `PoolMocker`
Now the actual interface can get either the `PoolMocker` object or the `Pool` and the respective methods be called.

Reply:
So may be we can think of using it as 
```
type PoolMocker struct{
...
}
```

Reply:
PoolMocker make more sense than MockPoolInfo(I will update all other places)

Reply:
Updated